src/restLevelClient/generateRestLevel.ts

```diff
@@ -76,9 +77,11 @@ export async function generateRestLevelClient() {
   generateIndexFile(project);
 
   generateTopLevelIndexFile(model, project);
-  if (generateTest) {
+  if (generateSample) {
     generateSampleEnv(project);
```

Comment:
@qiaozha I change the condition from `generateTest` to `generateSample`, could you help confirm any reason here to use `generateTest` to generate file `sample.env`?

Reply:
yeah, change to generateSample should be correct